auth-web/src/views/auth/home/DecideBusinessView.vue

```diff
@@ -43,11 +42,16 @@ import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
 export default class DecideBusinessView extends Vue {
   private readonly learnMoreUrl = 'https://smallbusinessbc.ca/article/how-to-choose-the-right-business-structure-for-your-' +
     'small-business/'
+  private readonly selectorWizardUrl = `https://entity-selection-dev.apps.silver.devops.gov.bc.ca`
   private readonly bulletPoints: Array<any> = [
-    { text: 'Decide which business structure is most appropriate for you: A few options are sole proprietorship, ' +
-      'partnership or incorporation.' },
-    { text: 'Each structure has a different legal and financial implications.',
-      linkText: 'Make sure a Benefit Company is right for you.'
+    {
+      text: `Decide which business structure is most appropriate for you, a few options are: a sole proprietorship,
```

Comment:
Run on sentence. Should be separated by either period or semi-colon.

Reply:
This copy was supplied by design.
Where are you thinking we need to break it up?

Reply:
"Decide which business structure is most appropriate for you. A few options are:"

or

"Decide which business structure is most appropriate for you -- a few options are:"

or

"Decide which business structure is most appropriate for you; a few options are:"

Reply:
Ok, i like the first too. Updated